packer/CentOS7/scripts/base.sh

```diff
@@ -2,4 +2,5 @@
 
 sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers
 
+yum -y update
```

Comment:
I agree about the speedup when you start from scratch. However, if you created the box a long time ago the `vagrant up` is still likely to require installing a lot of new package and a reboot. So I am +1 on this, just might not always help as much as you'd like.

Reply:
Yes, I know. And I didn't change the provisioning scripts; it is necessary of course to keep all boxes up-to-date. But just now I tried to create all VMs and it halted many times and took forever. With this change I rebuild the base box, updated it and simply ran `vagrant up`, grabbed a cup of coffee and 19 minutes later everything was up and running.